docs/EigenPods.md

```diff
@@ -3,7 +3,7 @@
 
 ## Overview
 
-This document explains *EigenPods*, the mechanism by which EigenLayer facilitates the restaking of native beacon chain ether.
+This document explains *EigenPods*, the mechanism by which EigenLayer facilitates the restaking of native beacon chain ether.  The EigenPods subprotocol allows entities that own validators that are apart of Ethereum's consensus to repoint their withdrawal credentials (explained later) to contracts in the EigenPods subprotocol that have certain mechanisms to ensure safe restaking.
```

Comment:
typo: `apart` => `a part` (missing space)

also eliminate the wording `that have certain mechanisms to ensure safe restaking`. This is filler and dangerously close to a a promise of safety

Reply:
fixed